### DISCUSSION TITLE
Need help with Nginx

### DISCUSSION THREAD

**Discussion by DarkSky5090** [nginx/nginx#160] [Q&A]
Hello, thanks for reading this
I have a raspberrypi and want to make a NAS server. I have installed all things, here is the command: wget -O - https://raw.githubusercontent.com/OpenMediaVault-Plugin-Developers/installScript/master/install | sudo bash
if i go to my pi's ip in my browser this shows up, imgae below
![Bild_2024-09-10_230550299](https://github.com/user-attachments/assets/b52efb56-4027-4fe2-beda-793e6dc825b8)
How do i fix this?

**Comment by ninaforsyth** [comment]
It looks like OpenMediaVault installs NGINX when you run the installer. I suggest you reach out to OpenMediaVault instead.
